nvm.psd1

```diff
@@ -12,7 +12,7 @@
     RootModule             = 'nvm.psm1'
 
     # Version number of this module.
-    ModuleVersion          = '2.1.0'
+    ModuleVersion          = '2.2.0'
```

Comment:
As it's a bug fix and not a feature, shouldn't this be a patch release?

Reply:
Yeah true. Rolled back to `2.1.1`